src/EventStore.Common/Log/log.config

```diff
@@ -8,26 +8,36 @@
   <variable name="VerboseLayout" value="[PID:${processid:padCharacter=0:padding=5}:${threadid:padCharacter=0:padding=3} ${date:universalTime=true:format=yyyy\.MM\.dd HH\:mm\:ss\.fff} ${level:padding=-5:uppercase=true} ${logger:padding=-20:fixedLength=true}] ${message}${onexception:${newline}EXCEPTION OCCURRED:${newline}${exception:format=tostring:innerFormat=tostring:maxInnerExceptionLevel=20}}"/>
   <variable name="ConsoleLayout" value="[${processid:padCharacter=0:padding=5},${threadid:padCharacter=0:padding=2},${date:universalTime=true:format=HH\:mm\:ss\.fff}] ${message}${onexception:${newline}EXCEPTION OCCURRED:${newline}${exception:format=message}}"/>
   <variable name="LogFileNameBase" value="${logsdir}/${shortdate:universalTime=true}/${environment:variable=EVENTSTORE_INT-COMPONENT-NAME}"/>
-  
-  <targets async="true">
-    <target name="fileLog" xsi:type="File" fileName="${LogFileNameBase}.log" layout="${VerboseLayout}"/>
-   
-    <target name="errorFileLog" xsi:type="FilteringWrapper" condition="length('${exception}')>0 || level==LogLevel.Error || level==LogLevel.Fatal">
-      <target xsi:type="File" fileName="${LogFileNameBase}-err.log" layout="${VerboseLayout}"/>
+
+  <targets>
+    <target name="asyncWrapper" xsi:Type="AsyncWrapper" overflowAction="Grow">
+      <target name="fileLog" xsi:type="File" fileName="${LogFileNameBase}.log" layout="${VerboseLayout}"/>
+    </target>
+
+    <target name="asyncWrapper" xsi:Type="AsyncWrapper" overflowAction="Grow">
```

Comment:
its pretty debatable whether consoles need an AsyncWrapper - AW is definitely not free